techniques/system/rudderApache/1.0/apache/logrotate.cf

```diff
@@ -0,0 +1,25 @@
+bundle agent system_apache_logrotate {
+  vars:
+    pass1.!duration_configured::
+      "logrotate_duration" string => "30";
+    pass1.duration_configured::
+      "logrotate_duration" string => "${rudder_parameters.log_duration}";
+
+    pass1::
+      "apache_logs"  data => parsejson('{
+                               "name": "${g.rudder_log}/apache2/*.log",
+                               "duration": "${logrotate_duration}",
+                               "create": "${rudder_syslog.file_mode} ${rudder_syslog.user_name} ${rudder_syslog.group_name}"
+                               "postrotate": "systemctl ${rudder_apache.service_name} reload > /dev/null || true"
+                             }');
+
+  classes:
+      "pass2" expression => "pass1";
+      "pass1" expression => "any";
+      "duration_configured" expression => isvariable("rudder_parameters.log_duration");
+
+  methods:
+    pass2::
+      "any" usebundle => _method_reporting_context("Apache configuration", "Logrotate");
```

Comment:
i don't see the Logrotate REPORTSKEYS in the metadata.xml

Reply:
Nice spot, will fixup today